src/test_encode_decode_aom.rs

```diff
@@ -324,7 +329,7 @@ fn encode_decode(
 
   let mut rec_fifo = VecDeque::new();
   for _ in 0..limit {
-    read_frame_batch(&mut ctx, &mut ra);
+    read_frame_batch(&mut ctx, &mut ra, w, h, ChromaSampling::Cs420);
```

Comment:
I think you need to rebase this onto the current master, because `encode_decode()` on both the AOM and dav1d ends now take a `chroma_sampling` argument and this will need to match.